src/perl/Permabit/Statistics/.gitignore

```diff
@@ -1 +0,0 @@
-Definitions.pm
```

Comment:
This file (src/perl/Permabit/Statistics/.gitignore) should really be part of the first commit since it's about not ignoring the newly-committed perl file.